src/couch_jobs/src/couch_jobs_fdb.erl

```diff
@@ -205,8 +206,11 @@ finish(#{jtx := true} = JTx0, #{jlock := <<_/binary>>} = Job, Data) when
             {error, halt}
     end.
 
+resubmit(JTx0, Job, NewSTime) ->
```

Comment:
Let's switch it such that the new `Data` parameter is at the end of the parameter list.

So the resubmit version look like:

```
resubmit(Tx, Job, SchedTime).
resubmit(Tx, Job, SchedTime, Data)
```